src/intrinsic.rs

```diff
@@ -260,7 +260,8 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a+'mir>: crate::MiriEvalContextExt<'a,
                 // However, this only affects direct calls of the intrinsic; calls to the stable
                 // functions wrapping them do get their validation.
                 // FIXME: should we check that the destination pointer is aligned even for ZSTs?
-                if !dest.layout.is_zst() { // nothing to do for ZST
+                if !dest.layout.is_zst() {
+                    // Nothing to do for ZST.
```

Comment:
This comment is about the case when the body is *not* entered, so it shouldn't be inside the body.

It could probably just be reworded or removed, anyway.